Marlin/src/lcd/marlinui.cpp

```diff
@@ -1035,11 +1035,13 @@ void MarlinUI::init() {
         uint8_t abs_diff = ABS(encoderDiff);
 
         #if ENCODER_PULSES_PER_STEP > 1
-          static int8_t lastEncoderDiff;
-          TERN_(HAS_TOUCH_SLEEP, if (lastEncoderDiff != encoderDiff) wakeup_screen());
-          lastEncoderDiff = encoderDiff;
+          #if HAS_TOUCH_SLEEP
+            static int8_t lastEncoderDiff);
+            if (lastEncoderDiff != encoderDiff) wakeup_screen());
+            lastEncoderDiff = encoderDiff);
```

Comment:
Do I miss something or are there closing brackets that don't belong there at the end of each line?

Reply:
you're correct. looks like they were forgotten.

Reply:
Good catch. I missed that too, and my config does not include those lines of code, so it compiled fine.